app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -575,7 +576,21 @@ private fun rateOnGooglePlayHeading(): UiObject {
 }
 
 private fun aboutFirefoxHeading(): UiObject {
-    settingsList().scrollToEnd(LISTS_MAXSWIPES)
+    for (i in 1..RETRY_COUNT) {
+        try {
+            settingsList().scrollToEnd(LISTS_MAXSWIPES)
+            assertTrue(
+                mDevice.findObject(UiSelector().text("About $appName"))
+                    .waitForExists(waitingTime),
+            )
+
+            break
+        } catch (e: AssertionError) {
+            if (i == RETRY_COUNT) {
+                throw e
+            }
```

Comment:
Do we need an else here to retry exiting out of settings and going in again?

Reply:
I think it's based on personal preference ?

The problem is that sometimes it only scrolls to "Rate on Google Play" button.

From my point of view, in case the scrolling fails, we already are in the "Settings" section, scrolled almost to the end, so just need to retry scrolling.

**Sure, I could add `else` and as you suggested, exit out of settings and re-attempt the scrolling.**

Please let me know @AaronMT  if I should do the changes, or leave it as it is. 🙂

Reply:
I would try with your current code, we could always change it if it starts failing.